### PR TITLE
Make scatter plot

### PR DESCRIPTION
**Pull request by BrandonPessman**
This adds a scatter plot that is currently parsing the big.csv and representing it. 
- Added Labels, Axis, and Points
- Reads in the big.csv
- Moved the rendering to the componentDidMount() to ensure it can find the right div to render the svg on

One thing we will still to look at is having buttons for choosing the range of data to represent. Currently we have to manually set the range in the code. Also, we can play around with how the ticks are represented and how many of them we want to have.

screen shot 2019-03-01 at 2 23 11 pm

Resolves #28

### CODE REVIEW

**Comment by BrandonPessman** [comment]
I once again screwed up the resolves, so its now branch 28 instead of 22.

**Comment by alex-stout** [comment]
No problem! haha. It took me a while to get that down too.

File: frontend/src/components/Home.js [hunk 1]
```diff
@@ -9,10 +9,9 @@ class Home extends Component {
         <center>
           <h1> Energy Dashboard </h1>
           <h4> A Powerful Dashboard for UWEC Energy Usage Visualization </h4>
-
+          <ScatterPlot></ScatterPlot>
```
Comment (alex-stout) [inline frontend/src/components/Home.js]: Should be `<ScatterPlot/>`

File: frontend/src/components/Home.js [hunk 2]
```diff
@@ -9,10 +9,9 @@ class Home extends Component {
...
         </center>
-        <div className="graph"></div>
-        <TestGraph></TestGraph>
+
```
Comment (alex-stout) [inline frontend/src/components/Home.js]: Extra whitespace.

**Comment by alex-stout** [comment]
This also brings up the question whether we want to use SCSS or not. It seems like you were doing some variable things in the CSS. Have you used SCSS/SASS before?

**Comment by BrandonPessman** [comment]
I haven’t looked into it before... I have noticed it in some projects though. I just have gotten use to doing variables for colors so you don’t have to replace everything when your designing. I’ll have to check it out

**Comment by alex-stout** [comment]
Check it out and let me know what you think. Head over to [codepen.io](https://www.codepen.io) to get an idea of what it is. You can chose SCSS instead of CSS in the options.

**Pull request merged**